doc/api/buffer.md

```diff
@@ -618,6 +618,10 @@ Returns the actual byte length of a string. This is not the same as
 [`String.prototype.length`] since that returns the number of *characters* in
 a string.
 
+_Note_ that for `'base64'`, this function assumes valid input. For strings that
```

Comment:
This isn’t specific to `base64`, it applies at least to `hex`, too:
```js
> Buffer.byteLength('aa  ', 'hex')
2
> Buffer.from('aa  ', 'hex').length
1
```

(I think it’s only those two encodings but I didn’t double-check.)